manifests/broker/service.pp

```diff
@@ -17,7 +17,7 @@
   $service_group = $::kafka::broker::service_group
 
   file { '/etc/init.d/kafka':
-    ensure  => present,
+    ensure  => file,
```

Comment:
If the content changes the file should be updated on system when using `ensure => present`